src/main/java/core/basesyntax/FileWork.java

```diff
@@ -1,7 +1,40 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
 public class FileWork {
+    private static final String STRING_START_WITH = "w";
+
     public String[] readFromFile(String fileName) {
-        return null;
+
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fileName))) {
+            StringBuilder stringBuilder = new StringBuilder();
+            String lineSeparator = System.lineSeparator();
+            String lineFromFile = bufferedReader.readLine();
+            while (lineFromFile != null) {
+                stringBuilder.append(lineFromFile).append(lineSeparator);
+                lineFromFile = bufferedReader.readLine();
+            }
+            String[] splitedStringBuilderResult = stringBuilder
+                    .toString()
+                    .toLowerCase()
+                    .split("\\s+|\\.|,|:|;|'|!|\\?|\r|\n");
```

Comment:
```suggestion
                    .split("\\s+|\\.|,|:|;|'|!|\\?");
```
\\s+ includes \r and \n

Reply:
done